File: fizz.php
<?php
    function fizzBuzz($fizzNum, $buzzNum) {
        $i = 1;
        while ($i <= 100) {
            $output = "";
            if ($i % $fizzNum === 0) {
                $output .= "Fizz";
            }
            if ($i % $buzzNum === 0) {
                $output .= "Buzz";
            }
            if (empty($output)) {
                $output = $i;
            }
            echo "$output\n";
            $i++;
        }
    }
    fizzBuzz(3, 5);
?>

File: fizz.py
#!/usr/bin/python3


def main():
    fizz_num = 3
    buzz_num = 5
    for number in range(1, 101):  # range - [x,y)
        output = ""
        if number % fizz_num == 0:
            output += "Fizz"
        if number % buzz_num == 0:
            output += "Buzz"
        if output == "":
            output = number
        print(output)


if __name__ == "__main__":
    main()
